test/specs/modules/Dropdown/Dropdown-test.js

```diff
@@ -620,6 +620,62 @@ describe('Dropdown Component', () => {
     })
   })
 
+  describe('removing items on backspace', () => {
+    let spy
+    beforeEach(() => {
+      spy = sandbox.spy()
+    })
+
+    it('does nothing without selected items', () => {
+      wrapperMount(<Dropdown {...requiredProps} multiple search onChange={spy} />)
+
+      // open
+      wrapper.simulate('click')
+
+      domEvent.keyDown(document, { key: 'Backspace' })
+
+      spy.should.not.have.been.called()
+    })
+    it('removes the last item when there is no search query', () => {
+      const value = _.map(options, 'value')
+      const expected = _.without(value, _.last(value))
+      wrapperMount(<Dropdown {...requiredProps} value={value} multiple search onChange={spy} />)
+
+      // open
+      wrapper.simulate('click')
+
+      domEvent.keyDown(document, { key: 'Backspace' })
+
+      spy.should.have.been.calledOnce()
+      spy.firstCall.args[1].should.deep.equal(expected)
```

Comment:
Hm, I wonder if it is a deep equality issue.  You could try using the getter form of `state` and then deep equal:

```js
wrapper
  .state('value')
  .should.deep.equal(expected)
```

If this still gives you troubles, don't worry about adding it, we can iterate on this.